### DISCUSSION TITLE
How to handle barchar height with negative values

### DISCUSSION THREAD

**Discussion by freudarian** [Abhinandan-Kushwaha/react-native-gifted-charts#624] [Q&A]
Hi,

i'm building a counter and register daily counts in the graph, bar chart vertical. When I set a height  it works good with the positive values, I see that the values on Y rescale.
But a user can also count negative values. But in this case the graph becomes bottomless.

I added a screenshot to illustrate. I can't find any solution within current options to avoid this. I only want to show the noOfSectionsBelowXAxis conditional IF values exist below 0. Also when I use noOfSectionsBelowXAxis the bars are cut off, and the horizontal line is always visible even if there are no negative values counted.

Is it a bug or a feature ? I would like to see that the height of the chart handles negative values the same as positive values, contained and limited within certain height, but I am not sure if I should create an issue for this. So I ask first my question here.

I expect behavior like this, change some value to -300, and you will understand what I mean by rescaling.
https://www.highcharts.com/demo/highcharts/column-negative

Please help :)
![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/71510589/8688f9af-1fef-4a1c-a008-0567428356b6)

**Comment by airowe** [comment]
@freudarian Did you ever find any solution for this?

**Reply by freudarian** [reply]
No actually the update didnt work as expected
